connect/connect-c7/worker-adapter-c7/src/main/java/io/miragon/miranum/connect/adapter/in/c7/worker/Camunda7WorkerAdapter.java

```diff
@@ -16,39 +14,35 @@
 import java.util.Map;
 import java.util.Objects;
 
-@Slf4j
 @RequiredArgsConstructor
-public class Camunda7WorkerAdapter implements BindWorkerPort {
+@Slf4j
```

Comment:
I will switch the logging to @log if it's the way to go. Do we have architecture decision records? It would be quite good to write decisions like the log framework somewhere down to avoid discussions like this in the future 😄

Reply:
Very valid point, we actually don't have or I'm not aware of any decision log.